tests/conftest.py

```diff
@@ -257,5 +267,33 @@ def build_header(path, method, role='submitter'):
         request.headers = dict(request.headers)
         auth.__call__(request)
         return request.headers
+
     return build_header
 
+
+@pytest.fixture(scope='session')
+def dictionary_setup():
+    def build_dict():
+        try:
+            session = requests.Session()
+            adapter = requests_mock.Adapter()
+            session.mount('s3', adapter)
+            json_dict = json.load(open(PATH_TO_SCHEMA_DIR + '/dictionary.json'))
+            adapter.register_uri('GET', 's3://test.com', json=json_dict, status_code=200)
+            resp = session.get('s3://test.com')
+
+            with patch('requests.get') as get_mocked:
+                get_mocked.return_value = resp
+                datadictionary = DataDictionary(url='fake_url')
+                sheepdog_blueprint = sheepdog.create_blueprint(
+                    'submission', datadictionary, gdcdatamodel.models
+                )
+
+                try:
+                    _app.register_blueprint(sheepdog_blueprint, url_prefix='/v0/submission')
+                except AssertionError:
+                    print('Blueprint is already registered!!!')
+        except Exception:
+            print('Exception')
```

Comment:
What kind of exceptions would this need to except?

Reply:
Fixed. Don't need any exception since status_code=200